pkg/qsfsd/qsfs.go

```diff
@@ -76,18 +73,20 @@ func setQSFSDefaults(cfg *zos.QuantumSafeFS) zstorConfig {
 func (q *QSFS) Mount(wlID string, cfg zos.QuantumSafeFS) (info pkg.QSFSInfo, err error) {
 	defer func() {
 		if err != nil {
-			if err := q.Unmount(wlID); err != nil {
-				log.Error().Err(err).Msg("error cleaning up after qsfs setup failure")
-			}
+			q.Unmount(wlID)
```

Comment:
Why removing the log line?

Reply:
The unmount doesn't return errors anymore it's modified to log errors. It's treated as a best-effort cleanup, because we can't do something about it when it fails. And I preferred to do the most cleanup that could be done (not return err early) even when one cleanup step fails.